### ISSUE TITLE
broken link for sylabs

### ISSUE THREAD

**Issue by shahzebsiddiqui** [easybuilders/easybuild#578]
While i was reading the easybuild container documentation https://easybuild.readthedocs.io/en/latest/Containers.html the following link was broken

```
%(bootstrap)s: bootstrap agent to use see also
https://www.sylabs.io/guides/latest/user-guide/definition_files.html#header
```

I assume it is the following link https://sylabs.io/guides/3.4/user-guide/definition_files.html#header but this is specific to version 3.4 embedded in link

**Comment by ocaisa** [comment]
Yeah, this is a moving target, not sure what the best approach is, the latest version is https://sylabs.io/guides/3.5/user-guide/definition_files.html#header

**Comment by shahzebsiddiqui** [comment]
My recommendation is to stick with the hardcoded version 3.4 or 3.5. If easybuild is performing some regression test against one of the singularity version, i would take that as the version name.  That could be one approach.

**Comment by boegel** [comment]
@gmkurtzer, @GodloveD, @vsoch: is there a generic URL we could use here, that always points to the latest version of the documentation?

**Comment by vsoch** [comment]
haha yeah! I actually opened this issue last summer https://github.com/sylabs/singularity-userdocs/issues/196 and although there aren't specific links to pages, you can get the root of "latest" via https://sylabs.io/guides/latest/user-guide (and same for the admin-docs).

**Comment by vsoch** [comment]
Given your example above to look at the header, I think the most you could do is direct the user to that link and then reference the section they should look at. The issue I linked isn't closed because I don't think they've figured out how to do it for latest (I'm not an employee of Sylabs so I'm not privy to how it's done / could help).
